docs/java-application-development/how-to-use-beacon-apis.md

```diff
@@ -1,14 +1,14 @@
-# How to Use New Beacon APIs
+# How to Use Beacon APIs
```

Comment:
```suggestion
# How to Use iBeacon&trade; APIs
```

for consistency?

Reply:
The term **Beacon** is a general API from the BT stack that has been implemented in several ways. IBeacon and Eddystone are two examples. So, the Kura API refers to the general BT API, not to the specific IBeacon implementation.